From 1ab87b9966a3adaad0a357adf158ab6e74d21ccb Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Wed, 8 Jan 2014 04:41:02 -0600
Subject: [PATCH 478/987] fsl_qbman: Fix the checkpatch warning type
 SPACE_BEFORE_TAB

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

WARNING:SPACE_BEFORE_TAB: please, no space before tabs

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Change-Id: I7298554a569ddf51b07760e87f9a1701706d8916
Reviewed-on: http://git.am.freescale.net:8181/7775
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Reviewed-by: Emilian Medve <Emilian.Medve@freescale.com>
---
 drivers/staging/fsl_qbman/qman_test_hotpotato.c | 2 +-
 include/linux/fsl_qman.h                        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_qbman/qman_test_hotpotato.c b/drivers/staging/fsl_qbman/qman_test_hotpotato.c
index 74d4a06..05db7ee 100644
--- a/drivers/staging/fsl_qbman/qman_test_hotpotato.c
+++ b/drivers/staging/fsl_qbman/qman_test_hotpotato.c
@@ -182,7 +182,7 @@ static dma_addr_t frame_dma;
 /* the main function waits on this */
 static DECLARE_WAIT_QUEUE_HEAD(queue);
 
-#define HP_PER_CPU 	2
+#define HP_PER_CPU	2
 #define HP_LOOPS	8
 /* 80 bytes, like a small ethernet frame, and bleeds into a second cacheline */
 #define HP_NUM_WORDS	80
diff --git a/include/linux/fsl_qman.h b/include/linux/fsl_qman.h
index d78a3b1b..f2968f9 100644
--- a/include/linux/fsl_qman.h
+++ b/include/linux/fsl_qman.h
@@ -242,7 +242,7 @@ struct qm_sg_entry {
 		};
 	};
 	u32 extension:1;	/* Extension bit */
-	u32 final:1; 		/* Final bit */
+	u32 final:1;		/* Final bit */
 	u32 length:30;
 	u8 __reserved2;
 	u8 bpid;
-- 
1.9.1

